[2025-02-08T15:03:10.044+0200] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T15:03:10.058+0200] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: patient-monitoring.add_new_patient scheduled__2025-02-08T13:02:00+00:00 [queued]>
[2025-02-08T15:03:10.066+0200] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: patient-monitoring.add_new_patient scheduled__2025-02-08T13:02:00+00:00 [queued]>
[2025-02-08T15:03:10.066+0200] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-08T15:03:10.082+0200] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): add_new_patient> on 2025-02-08 13:02:00+00:00
[2025-02-08T15:03:10.089+0200] {standard_task_runner.py:72} INFO - Started process 61919 to run task
[2025-02-08T15:03:10.096+0200] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'patient-monitoring', 'add_new_patient', 'scheduled__2025-02-08T13:02:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/real_time_patient_data_update.py', '--cfg-path', '/tmp/tmph8z7ouoo']
[2025-02-08T15:03:10.098+0200] {standard_task_runner.py:105} INFO - Job 69: Subtask add_new_patient
[2025-02-08T15:03:10.156+0200] {task_command.py:467} INFO - Running <TaskInstance: patient-monitoring.add_new_patient scheduled__2025-02-08T13:02:00+00:00 [running]> on host munyah-ThinkPad-X1-Yoga-1st
[2025-02-08T15:03:10.264+0200] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='munyah' AIRFLOW_CTX_DAG_ID='patient-monitoring' AIRFLOW_CTX_TASK_ID='add_new_patient' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T13:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-08T13:02:00+00:00'
[2025-02-08T15:03:10.266+0200] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-08T15:03:10.266+0200] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-08T15:03:10.266+0200] {logging_mixin.py:190} INFO - Current task name:add_new_patient state:running start_date:2025-02-08 13:03:10.058448+00:00
[2025-02-08T15:03:10.266+0200] {logging_mixin.py:190} INFO - Dag name:patient-monitoring and current dag run status:running
[2025-02-08T15:03:10.267+0200] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T15:03:10.300+0200] {real_time_patient_data_update.py:80} INFO - New Patient Added: {'Patient_ID': 'PAT-00051', 'Age': 0, 'Gender': 'Male', 'Height': 203.57, 'Weight': 156.76, 'Temperature': 36.5, 'Blood_Pressure': '126/64', 'Cholesterol_Level': 'Borderline', 'Blood_Sugar_Level': 255, 'Heart_Rate': 35, 'BMI': 20.33, 'Previous_Medical_History': 'No significant history', 'Comorbidities': 'None', 'Smoking_Status': 'Yes', 'Alcohol_Consumption': 'No', 'Physical_Activity': 'Low', 'Last_Visit_Date': '2024-06-30', 'Genomic_Info': 'Normal', 'Diagnosis': 'No specific diagnosis', 'Hospitalization': 'No'}
[2025-02-08T15:03:10.301+0200] {real_time_patient_data_update.py:81} INFO - Total Patients After Addition: 51
[2025-02-08T15:03:10.311+0200] {base.py:84} INFO - Retrieving connection 'patients_data'
[2025-02-08T15:03:10.318+0200] {http.py:204} ERROR - HTTP error: INTERNAL SERVER ERROR
[2025-02-08T15:03:10.319+0200] {http.py:205} ERROR - {
  "error": "Failed to process the request."
}

[2025-02-08T15:03:10.319+0200] {real_time_patient_data_update.py:94} ERROR - Error while adding patient to the database: 500:INTERNAL SERVER ERROR
[2025-02-08T15:03:10.320+0200] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T15:03:10.330+0200] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T15:03:10.331+0200] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=patient-monitoring, task_id=add_new_patient, run_id=scheduled__2025-02-08T13:02:00+00:00, execution_date=20250208T130200, start_date=20250208T130310, end_date=20250208T130310
[2025-02-08T15:03:10.344+0200] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-08T15:03:10.345+0200] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-08T15:03:10.345+0200] {logging_mixin.py:190} INFO - Dag name:patient-monitoring queued_at:2025-02-08 13:03:01.507570+00:00
[2025-02-08T15:03:10.346+0200] {logging_mixin.py:190} INFO - Task hostname:munyah-ThinkPad-X1-Yoga-1st operator:PythonOperator
[2025-02-08T15:03:10.391+0200] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T15:03:10.415+0200] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T15:03:10.416+0200] {local_task_job_runner.py:245} INFO - ::endgroup::
